web/withdrawals_controller.go

```diff
@@ -18,29 +19,50 @@ type WithdrawalsController struct {
 
 var naz = assets.NewLink(1)
 
+// abortOrWarn aborts the web request represented by c, by registering an error
+// (err), or logs a warning if that fails for some reason.
+func abortOrWarn(c *gin.Context, err error) {
+	if errErr := c.AbortWithError(500, err); errErr != nil {
```

Comment:
The error returned by `c.AbortWithError` isn't a new error as a result of being unable to abort, but rather the parsed error you passed in. Therefore it will return an error on success, causing us to erroneously log a warning:

https://github.com/gin-gonic/gin/blob/521d06c81d5e49a004473daa81b71e1d34fe7a05/context.go#L145-L174

I would remove this whole method and stick with the existing pattern of `c.AbortWithError` in the controller.

Reply:
Hah, I just assumed `AbortWithError` was following go's standard error-handling. Thanks.

Reply:
Fixed in 98e1f5d